internal/provider/cluster_resource.go

```diff
@@ -1363,6 +1377,11 @@ func loadClusterToTerraformState(
 			PrivateNetworkVisibility: types.BoolValue(clusterObj.GetNetworkVisibility() == client.NETWORKVISIBILITYTYPE_PRIVATE),
 			CidrRange:                types.StringValue(clusterObj.CidrRange),
 		}
+		if plan != nil && plan.DedicatedConfig != nil {
+			// We want to make sure to set this bool to false if it was
+			// omitted entirely.
+			state.DedicatedConfig.SupportPhysicalClusterReplication = types.BoolValue(plan.DedicatedConfig.SupportPhysicalClusterReplication.ValueBool())
+		}
```

Comment:
This part is somewhat confusing.  I wonder if we could make it a bit better.  I think partially it's confusing because defaulting to false is mentioned in the comment and then the value from the plan is chosen. Presumably the value could be true?   

The other thing that I think is confusing is about when the plan would be nil.  Inspecting the callers reveals that on a Read the state is passed in as the plan. I think that is the only case where it might be nil? But still that seems a bit strange.   If you've figured this out during development it would be good to leave a more detailed comment about this situation.  

I'm also wondering if we can incorporate it into the setting of `state.DedicatedConfig` above instead of after the fact.